docs/integrations/metabase.rst

```diff
@@ -0,0 +1,107 @@
+.. _integrations-metabase:
+
+Visualize data with Metabase
+============================
+
+This tutorial introduces `Metabase`_, another visualization tool that can help
+you analyze and use your data to its full potential.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. _metabase-prereqs:
+
+Prerequisites
+-------------
+
+First, you will need a running cluster. You can use Metabase with both
+:ref:`Cloud clusters <cluster-deployment-stripe>` and :ref:`Edge clusters
+<edge>`.
+
+To use Metabase, you must have an existing data set in your CrateDB cluster.
+Feel free to use the sample dataset  available in the Cloud Console or
+import your own data similarly to how it's done :ref:`in this how-to
+<cloud-howtos:import-data>`.
+
+.. _integration-metabase-config:
+
+Initial configuration
+---------------------
+
+Metabase offers both cloud version and local installation. Whichever you
+choose, the first step will be adding your CrateDB cluster as a new database.
+To do that, go to the ``Admin Settings`` -> ``Setup``, and choose 
+the ``Add a database`` option.
+
+.. image:: ../_assets/img/metabase-add-database.png
+   :alt: Add new database
+
+Database configuration is relatively simple, these are the necessary fields:
+
+- Database type (PostgreSQL)
+- Display name
+- Host (the URI of your cluster)
+- Database name
+- Username
+- Password
+
+.. NOTE::
+
+    If you choose to use Metabase with a Cloud cluster, also select the "Use
```

Comment:
SSL is pretty normal in all clusters, perhaps we should rephrase this to "Make sure you select "Use a secure connection (SSL)" option, unless your cluster is not configured for SSL".